### ISSUE TITLE
Multiple instances

### ISSUE THREAD

**Issue by FrostBy** [Arcana/node-dota2-bot#7]
Hi!
Would you please add a multiple bot instances support?
For example, i want to create a ladder for my project, but the bot works only with the single lobby at the single moment, while this lobby is not started. So the sentry file will be overwritten. Maybe some param for the constructor(sentry suffix or smth else)?

Real case from the project:
I create a connection to RabbitMQ and listen commands from it. If a bot is busy(scans profiles or waits for lobby launch), I launch a bot with another login/password pairs. If all available bots are busy, i will add a task to schedule. After a task is done, I post another task to RabbitMQ for PHP server.

Maybe you can provide me an example, how can I implement it in right way with the current implementation? Unfortunately, i'm new in NodeJs. 

For now I will use a fork of this implementation with my changes.

**Comment by Crazy-Duck** [comment]
Yeah, that's a feature I haven't implemented yet. I'm not sure when I'll get around to do it, but if you know what to do, feel free to create a PR. The best way to go about it would probably to turn the sentry file into a csv where you keep username;sentry pairs I think

**Comment by FrostBy** [comment]
If it will be possible and not so critical (commercial product), i'll create a separate repo with my own bot(RabbitMQ + Laravel job builder + balancer + chat commands listener) and send a link into the comments.
